packages/atclient/include/atclient/atkeys_file.h

```diff
@@ -51,6 +51,15 @@ typedef struct atclient_atkeys_file {
   uint8_t _initialized_fields[1];
 } atclient_atkeys_file;
 
+/**
+ * @brief Gets the default path for the atKeys file for a given atsign
+ *
+ * @param atsign The atsign to get the default path for (with the '@')
+ * @return char* The default path to the atKeys file. The caller is responsible for freeing this memory.
+ *                Returns NULL if memory allocation fails or if atsign is NULL.
+ */
+char *atkeys_file_get_default_path(const char *atsign);
+
```

Comment:
A similar function was in atauth, but it belongs in atclient, with the other atkeys file functions.